src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -76,7 +77,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $routes = $this->router->getRouteCollection();
 
         if ($name) {
-            if (!$route = $routes->get($name)) {
+            if (!($route = $routes->get($name)) && $matchingRoutes = $this->findRouteNameContaining($name, $routes)) {
+                $default = 1 === count($matchingRoutes) ? $matchingRoutes[0] : null;
+                $name = $io->choice('Select one of the matching routes to display its information', $matchingRoutes, $default);
```

Comment:
I think the `to display its information` is not needed.

Reply:
I removed this part.